No electron experience => unable to start project

Hi there,
I'm happy to see that this repo has become active again! It's been in my starred repos for a while and it made me want to give it a go. I've never done anything with electron though, and that's probably why I'm having trouble here. But as I'm sure there are many other electron beginners out there, it might be worth it to make the README a little more comprehensive.

Here's my issue: I can't start the project at all.
Question: what am I doing wrong? 

### What I've tried

With electron installed globally, following README.md instructions (empty index.html and copy/pasted myApp.js):
```bash
$ electron myApp.js 

App threw an error during load
[some path]/myApp.js:1
(function (exports, require, module, __filename, __dirname, process, global) { import { menubar } from 'menubar';
                                                                               ^^^^^^
SyntaxError: Unexpected token import
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at loadApplicationPackage (/usr/local/lib/node_modules/electron-prebuilt/dist/Electron.app/Contents/Resources/default_app.asar/main.js:280:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/electron-prebuilt/dist/Electron.app/Contents/Resources/default_app.asar/main.js:322:5)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
```

In examples/hello-world, with 6.0.3 release, and after a `yarn install` at root. 
```bash
$ yarn install
$ electron .
App threw an error during load
Error: Cannot find module '../..'
Require stack:
- [some path]/menubar-6.0.3/examples/hello-world/main.js
- [some path]/menubar-6.0.3/examples/node_modules/electron/dist/Electron.app/Contents/Resources/default_app.asar/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:659:15)
    at Function.Module._resolveFilename ([some path]/menubar-6.0.3/examples/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/common/reset-search-paths.js:43:12)
    at Function.Module._load (internal/modules/cjs/loader.js:577:27)
    at Module.require (internal/modules/cjs/loader.js:715:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> ([some path]/menubar-6.0.3/examples/hello-world/main.js:1:21)
    at Module._compile (internal/modules/cjs/loader.js:808:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:820:10)
    at Module.load (internal/modules/cjs/loader.js:677:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:609:12)
```

### Specifications

- Menubar version: master and 6.0.3
- Platform: macOS 10.14.5
- Node.js version: 12.3.1

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,5 +12,4 @@ install:
   - yarn install
 script:
   - yarn lint
-  - yarn build
   - yarn test

--- a/package.json
+++ b/package.json
@@ -37,6 +37,7 @@
     "prebuild": "rimraf lib/",
     "build": "tsc",
     "lint": "tslint --project .",
+    "postinstall": "yarn build",
     "test": "jest"
   },
   "types": "lib/index.d.ts",

